dev-python/qdarkstyle/qdarkstyle-2.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+MY_PN=QDarkStyleSheet
+
+DESCRIPTION="A dark style sheet for QtWidgets application"
+HOMEPAGE="https://github.com/ColinDuquesnoy/QDarkStyleSheet"
+SRC_URI="https://github.com/ColinDuquesnoy/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc develop examples test"
+
+RDEPEND=">=dev-python/helpdev-0.6.2[${PYTHON_USEDEP}]
+	develop? ( dev-python/qtsass[${PYTHON_USEDEP}]
```

Comment:
Ok, so, this USE flag has really no use in this ebuild. How does it detect when to use qtsass and when not to? Is it conditional? Then I don't really like the name `develop`, it doesn't tell much what it does especially since it just pulls qtsass. Also, https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs having IUSE without having other function than pulling RDEPEND in the ebuild is banned. You can use `optfeature` instead and remove `develop` altogether. In case it's really **not** needed here, but is optional runtime dependency.

Reply:
Fixed, I did a search through the source files and qtsass is only used by qdarsktyle.utils, not the actual stylesheet.